frontend/src/httpLayers/signout.http.js

```diff
@@ -0,0 +1,26 @@
+const { default: wofApi } = require("./wofApi");
```

Comment:
It is unnecessary. Below we have it imported

Reply:
Mhm, I must have missed one XD

Reply:
Removed